src/commands/publish/mod.rs

```diff
@@ -268,7 +269,7 @@ fn validate_target_required_fields_present(target: &Target) -> Result<(), failur
         {
             missing_fields.push("zone_id")
         };
-        if target.route.as_ref().unwrap_or(&"".to_string()).is_empty() {
+        if route.is_empty() {
```

Comment:
should probably check if `.routes()` is empty, no? This is part of the reason I'm wondering if we should remove `route` from `target` and replace it with `routes`. But yeah if we still want the method we should use it here

Reply:
yes, i'll be coming through and cleaning up references to `target.route` in a later PR. good catch!

Reply:
this would also be more like `for route in routes ...`